Maman11/Q2/my_suffix.c

```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+#define STRING_LENGTH 127
+
+int suffix(char str[], char c){
+    bool read = false;
+    int suffix = 0, len = strlen(str), i = 0;
+    while (str[i]){
+        if (c == str[i]){
+            read = true;
+        }
+        if (read){
```

Comment:
If you set read to true you can just do the following:

if (char is equal)
   do the prints and count ++


this can be the only if in the loop